QualityUnit/postaffiliatepro-theme
# [726] Buttons in Header and Hero section

**Changes proposed in this Pull Request**
New button "Book a consult" in header.
Smaller space between hero buttons.

**Testing instructions**
Check Header in all resolutions and all languages.
Check gap between hero buttons.

**Checklist**

Close QualityUnit/web-issues#726
